libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -696,30 +691,7 @@ constructor(
                 .navigatorPredictionMillis(DEFAULT_NAVIGATOR_PREDICTION_MILLIS)
                 .distanceFormatter(distanceFormatter)
 
-            val tilesUri = URI("https://api.mapbox.com")
-            // Latest version available https://api.mapbox.com/route-tiles/v1/versions?access_token=
-            // as of 05/18/2020
-            val tilesVersion = "2020_02_02-03_00_00"
-            val tilesDir = if (tilesUri.toString().isNotEmpty() && tilesVersion.isNotEmpty()) {
```

Comment:
We lost this check. Is it safe?

Reply:
Yeah fair question. Will reproduce the behavior the failure case in OnboardRouterEndpointOptionsTest

but defaultOptions shouldn't have issue, we may want to crash if it does

Reply:
added tests, builder will fail to build